bloom.js

```diff
@@ -87,12 +74,32 @@ JSBloom.filter = function(items, target_prob) {
     }
 
     checkEntry = function(str) {
+        var index, extra_indices
```

Comment:
Pedantic but keeping with the existing code format would be best (terminate each line with ;).